solutions/vmware/migration/ansible/vm-metadata-operations/roles/setup_prerequisites/tasks/install_docker.yml

```diff
@@ -0,0 +1,47 @@
+---
+- name: Install yum utils
+  yum:
+    name: yum-utils
+    state: latest
+
+- name: Add Docker repo
+  get_url:
+    url: https://download.docker.com/linux/centos/docker-ce.repo
+    dest: /etc/yum.repos.d/docer-ce.repo
+
+- name: Install Docker and Docker Compose
+  yum:
+    name: 
+      - "docker-ce-{{ docker_version }}"
+      - "docker-ce-cli-{{ docker_version }}"
+      - "containerd.io"
+      - "docker-compose-plugin"
```

Comment:
@gauravchaudhari02 - `docker-compose-plugin` package does not install `docker-compose` command which we hase used in scripts. Refer - https://github.com/Seagate/cortx-re/blob/main/docker/cortx-deploy/build.sh#L126 

```bash
[root@ssc-cicd-4094 ~]# yum list docker-compose-plugin
Updating Subscription Management repositories.
EPEL 8 - Everything                                                                                                                                       45 kB/s | 2.3 kB     00:00
Rocky Linux 8 - AppStream                                                                                                                                 53 kB/s | 2.6 kB     00:00
Docker-CentOS-8                                                                                                                                           38 kB/s | 2.0 kB     00:00
Rocky Linux 8 - PowerTools                                                                                                                                50 kB/s | 2.6 kB     00:00
puppet6-el8                                                                                                                                               34 kB/s | 1.7 kB     00:00
Rocky Linux 8 - Extras                                                                                                                                    41 kB/s | 2.0 kB     00:00
Rocky Linux 8 - BaseOS                                                                                                                                    44 kB/s | 2.3 kB     00:00
Rocky Linux 8 - ResilientStorage                                                                                                                          46 kB/s | 2.3 kB     00:00
Kubernetes-EL7                                                                                                                                            43 kB/s | 2.0 kB     00:00
Installed Packages
docker-compose-plugin.x86_64                                                                2.14.1-3.el8                                                                @docker-ce-stable
[root@ssc-cicd-4094 ~]# docker-compose
-bash: docker-compose: command not found
[root@ssc-cicd-4094 ~]#
```

We either need to update scripts or use https://docs.docker.com/compose/install/linux/#install-the-plugin-manually

Reply:
Added suggested way to install docker-compose